test/unit/instrumentation/prisma-client.test.js

```diff
@@ -43,7 +43,10 @@ test('PrismaClient unit tests', (t) => {
     let PrismaClient
     if (semver.gte(version, '4.11.0')) {
       PrismaClient = function PrismaClient() {
```

Comment:
I know that this is unrelated to your changes, but since these are new files I'm bringing it up, there are two eslint warnings in here we might want to fix:
```
/Users/jlopatta/projects/node-newrelic/test/unit/instrumentation/prisma-client.test.js
  45:31  warning  'PrismaClient' is already declared in the upper scope on line 43 column 9  no-shadow
  53:31  warning  'PrismaClient' is already declared in the upper scope on line 43 column 9  no-shadow
```

Reply:
yea i saw that, i can change it's kinda annoying because it doesn't affect anything